python/src/Sample.i

```diff
@@ -786,6 +786,26 @@ Sample(PyObject * pyObj)
   return new OT::Sample( OT::convert< OT::_PySequence_, OT::Sample>(pyObj) );  
 }
 
+static Sample BuildFromSequence(PyObject * pyObj, const OT::UnsignedInteger size, const OT::UnsignedInteger dimension)
+{
```

Comment:
-1
when d=1 (like I suppose this will be in most cases) simply writes `ot.Sample([[x] for x in L])` !
even when d>1 this will be `ot.Sample([L[i:i+d] for i in range(0,len(L),d)])` which is not uber difficult
so imho no need for this function when the solution is built-in the language
worst case a simple example would be enough

Reply:
I move this (for discussion) into a new PR in order to not freeze this one